lib/tests/streamlit/runtime/app_session_test.py

```diff
@@ -764,41 +764,20 @@ async def test_new_session_message_includes_fragment_ids(self):
             event=ScriptRunnerEvent.SCRIPT_STARTED,
             page_script_hash="",
             fragment_ids_this_run=["my_fragment_id"],
-            clear_forward_msg_queue=False,
         )
 
         # Yield to let the AppSession's callbacks run.
         await asyncio.sleep(0)
 
         sent_messages = session._browser_queue._queue
         assert len(sent_messages) == 2  # NewApp and SessionState messages
-        session._clear_queue.assert_not_called()
+        session._clear_queue.assert_called_once()
 
         new_session_msg = sent_messages[0].new_session
         assert new_session_msg.fragment_ids_this_run == ["my_fragment_id"]
 
         add_script_run_ctx(ctx=orig_ctx)
 
-    async def test_clears_forward_msg_queue_by_default(self):
```

Comment:
we don't need this test anymore because the parameter `clear_forward_msg_queue` is removed on the `_handle_scriptrunner_event_on_event_loop` function (triggered by `_on_scriptrunner_event`)